### PR TITLE
📈 track with segment

### LINKED ISSUE #1282: analytics: segment + mixpanel

### PR DESCRIPTION
**Pull request by sebipap**
1. added tracking for
  * NavBar
  * Markets Basic
  * Markets Advanced + Operations Modal + Market Pages

2. removed `react-ga4` and related logic

closes #1282

### CODE REVIEW

**Review by jgalat — APPROVED** [review]
lgtm
This sort of resembles how we used analytics when universal analytics was available (imho, a better solution than GA4). I definitely prefer explicitely having the fields for the tracked event rather than calling a callback with implicit fields that are being tracked from the react context. That brought a lot of trouble to us having to adapt each context to live in the outermost of the app just for the sake of having one analytics callback.

File: components/AdvancedSwitch/index.tsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
-import React, { FC } from 'react';
+import React, { FC, useCallback } from 'react';
 import { Box, BoxProps, Typography } from '@mui/material';
 import { useCustomTheme } from 'contexts/ThemeContext';
 import { useTranslation } from 'react-i18next';
 import Switch from 'components/Switch';
+import { track } from '../../utils/segment';
```
Comment (franm91) [inline components/AdvancedSwitch/index.tsx:6]: same here

**Review by franm91 — DISMISSED** [review]

File: hooks/useApprove.ts [hunk 2]
```diff
@@ -150,19 +147,28 @@ function useApprove({
           break;
         }
       }
+      track('TX Signed', {
+        operation,
+        hash,
+        method: 'approve',
+        contractName: 'Market',
+        spender,
+        qty: Number(quantity),
```
Comment (jgalat) [inline hooks/useApprove.ts:156]: I see below that you are just using the bigint for `qty`, should this also be the case?

Another question, should we consider formatting this numbers with their corresponding decimals?

**Review by jgalat — DISMISSED** [review]
lgtm, minor comment

**Review by jgalat — APPROVED** [review]

File: hooks/useApprove.ts (hunk 2 above)
Comment (sebipap) [reply hooks/useApprove.ts]: we'll send the values in floating point

**Review by franm91 — APPROVED** [review]

**Pull request merged**